health-services/project/src/main/java/org/egov/project/config/ProjectConfiguration.java

```diff
@@ -189,4 +189,29 @@ public class ProjectConfiguration {
     @Value("${project.staff.attendance.topic}")
     private String projectStaffAttendanceTopic;
 
+
+    // closed household task
+    @Value("${project.user.action.task.kafka.create.topic}")
+    private String createUserActionTaskTopic;
+
+    @Value("${project.user.action.task.consumer.bulk.create.topic}")
+    private String bulkCreateUserActionTaskTopic;
+
+    @Value("${project.user.action.task.kafka.update.topic}")
+    private String updateUserActionTaskTopic;
```

Comment:
remove task for useraction and location topic

Reply:
done